shared/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -607,4 +607,16 @@
     <string name="paywall_pending_purchase">Paid functionality will become available after success payment</string>
     <string name="paywall_subscription_sync_description">The app is updating. Please wait a moment</string>
 
+    <!--Manage subscription-->
+    <string name="manage_subscription_screen_title">Subscription</string>
+    <string name="manage_subscription_plan_title">Your current plan:</string>
+    <string name="manage_subscription_mobile_only">Hyperskill Mobile only</string>
```

Comment:
I suppose we don't need to hardcode subscription display name, I'm sure that SDK must provide it.

_For example in Google Play / App Store we can update product name (subscription display name) and make new submission without app resubmit, so when user will buy subscription it will see different names in the app and in the system purchase UI._

Reply:
@ivan-magda 
I guess, we need to load the subscription name on the Paywall screen also.
I suggest doing it in a separate task.

Reply:
@XanderZhu OK